site/en/workflows.md

```diff
@@ -433,17 +445,23 @@ By now the image should be well prepared for segmentation.
 ### Step 7: Region segmentation
 
 <!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-region-segmentation.md|sed '$d' -->
-
 In this processing step, an (optimized) document image is taken as an input and the
 image is segmented into the various regions, including columns.
 Segments are also classified, either coarse (text, separator, image, table, ...) or fine-grained (paragraph, marginalia, heading, ...).
 
-**Note:** If you use `ocrd-tesserocr-segment-region`, which uses only bounding boxes instead of polygon coordinates,
-then you should post-process via `ocrd-segment-repair` with `plausibilize=True` to obtain better results without large overlaps.
+**Note:** If you use `ocrd-tesserocr-segment-region`, which uses only bounding
+boxes instead of polygon coordinates, then you should post-process via
+`ocrd-segment-repair` with `plausibilize=True` to obtain better results without
+large overlaps. Alternatively, consider using the all-in-one [`ocrd-tesserocr-recognize`](#step-x-multistep)
+processor that can do region segmentation, line segmentation and text recognition
+in one step and can query tesseract for the generally more precise polygon region outlines
+instead of more coarse bounding boxes.
```

Comment:
```suggestion
large overlaps. _Alternatively_, consider using the all-in-one capabilities of [`ocrd-tesserocr-segment` and `ocrd-tesserocr-recognize`](#step-x-multistep), which can do region segmentation and line segmentation (and optionally also text recognition) in one step by querying Tesseract's internal iterator (accessing the more precise polygon outlines instead of just coarse bounding boxes with lots of hard-to-recover overlap).
```